collect_app/src/test/java/org/odk/collect/android/database/FormDatabaseMigratorTest.java

```diff
@@ -63,11 +64,15 @@ public void teardown() {
 
     @Test
     public void onUpgrade_fromVersion11() {
+        int oldVersion = 11;
+        assertTrue(oldVersion < DatabaseConstants.FORMS_DATABASE_VERSION);
```

Comment:
I really like this approach. In files like this, tests are pretty much always copied, pasted and edited so the chances of "virally" carrying over the check and it ending up being modified correctly feels pretty high here. A step further would be set this all up as a parameterized test, but I think this is good enough for now!